earthmover/nodes/source.py

```diff
@@ -266,13 +265,29 @@ def __get_skiprows(config: 'YamlMapping'):
             _header_rows = config.get('header_rows', 1)
             return int(_header_rows) - 1  # If header_rows = 1, skip none.
 
+        def __read_fwf(file: str, config: 'YamlMapping'):
```

Comment:
We should define any helper methods like these outside the `__get_read_lambda()` helper.

I understand the purpose of what we're doing here, but it does not spark joy. We are defining our own filespec for documenting FWF headers. There are a couple of improvements I might suggest:
- Make the columns of the file name-agnostic. (However, how do we handle that optional column`?)
- Very clearly and forcefully define the filespec in the README, and tell users explicitly how to use it.